modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -1264,76 +1264,123 @@ func (suite *KeeperTestSuite) TestChanUpgradeCancel() {
 }
 
 func (suite *KeeperTestSuite) TestWriteUpgradeCancelChannel() {
-	suite.SetupTest()
+	var (
+		path         *ibctesting.Path
+	)
```

Comment:
I think the linter might complain about this 🤔 it might request to remove the parentheses